src/ripple/rpc/impl/Role.cpp

```diff
@@ -108,14 +109,7 @@ requestInboundEndpoint (Resource::Manager& manager,
 boost::string_view
 forwardedFor(http_request_type const& request)
 {
-    auto it = request.find("X-Forwarded-For");
-    if (it != request.end())
-    {
-        return boost::beast::http::ext_list{
-            it->value()}.begin()->first;
-    }
-
-    it = request.find("Forwarded");
+    auto it = request.find(boost::beast::http::field::forwarded);
```

Comment:
you might want to `#include` the header that defines the fields, just for completeness.

Reply:
fixed